cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -107,64 +106,64 @@ func makeMachineOwner(i int, replicaCount int, annotations map[string]string, ow
 }
 
 func TestControllerFindMachineByID(t *testing.T) {
-	controller, stop := mustCreateTestController(t, testControllerConfig{})
-	defer stop()
-
-	testMachine := &v1beta1.Machine{
-		ObjectMeta: v1.ObjectMeta{
-			Name:      "test",
-			Namespace: "test-namespace",
-		},
-	}
-
-	// Verify machine count starts at 0.
-	machines, err := controller.machineInformer.Lister().Machines(v1.NamespaceAll).List(labels.Everything())
-	if err != nil {
-		t.Fatalf("unexpected error, got %v", err)
-	}
-	if len(machines) != 0 {
-		t.Fatalf("expected 0 machines, got %d", len(machines))
-	}
-
-	controller.machineInformer.Informer().GetStore().Add(testMachine)
-
-	// Verify machine count goes to 1.
-	machines, err = controller.machineInformer.Lister().Machines(v1.NamespaceAll).List(labels.Everything())
-	if err != nil {
-		t.Fatalf("unexpected error, got %v", err)
-	}
-	if len(machines) != 1 {
-		t.Fatalf("expected 1 machine, got %d", len(machines))
+	type testCase struct {
+		description    string
+		name           string
```

Comment:
Nit: add comment the `name` is used as a namespace for `newMachineSetTestObjs`. Thus, it needs to compliant with namespace naming.

Reply:
`tc.name` is not passed to `newMachineSetTestObjs()`. What is passed is `t.Name()` which is the name of this function under Test. And that, as you point out is not compliant as it uses upper case characters. Given that we have just merged a number of PRs doing exactly the same thing can I follow up and fix all these in one go. One solution would be to do `lowercase(t.Name())`.

Reply:
> Nit: add comment the `name` is used as a namespace for `newMachineSetTestObjs`. Thus, it needs to compliant with namespace naming.

I misread this. for `name` you meant `t.Name()`. See - https://github.com/openshift/kubernetes-autoscaler/pull/61/files#r266810534